lib/control/v1/authenticate.js

```diff
@@ -32,7 +36,7 @@ function Authenticate(app, server, vtoken) {
     fetch.metadata,
 
     function (req, res, next) {
-      token.create(req, res, next, vtoken);
+      token.create(req, res, next, Config.get('vault'));
```

Comment:
I want to get all of the Vault object here, as I am passing in the hostname, port, and token.